kubernetes/samples/charts/apache/default-sample.md

```diff
@@ -0,0 +1,38 @@
+# Apache Load Balancer default sample
+## Configure Apache Webtier as Load Balancer for WLS Domains
+In this section we will demonstrate how to use Apache webtier to handle traffic to backend WLS domains.
+
+### 1. Install WebLogic Server Domain
+Now we need to prepare some backends for Apache to do load balancing.
+
+Create a WebLogic Server domain:
```

Comment:
Can refer the create-weblogic-domain/domain-home-on-pv sample as an example.

Reply:
Yes we need to standarize our samples to all point to other samples in our project.  Please point to sample of domain in pv.